Objects/codeobject.c

```diff
@@ -3330,12 +3330,29 @@ _PyCodeArray_New(Py_ssize_t size)
     return arr;
 }
 
+// Get the underlying code unit, leaving instrumentation
+static _Py_CODEUNIT
+deopt_code_unit(PyCodeObject *code, int i)
+{
+    _Py_CODEUNIT *src_instr = _PyCode_CODE(code) + i;
+    _Py_CODEUNIT inst = {
+        .cache = FT_ATOMIC_LOAD_UINT16_RELAXED(*(uint16_t *)src_instr)};
+    int opcode = inst.op.code;
```

Comment:
Initializing .cache and then accessing .op feels a bit icky. Do we do this (assuming .cache covers the whole of the _Py_CODEUNIT union) elsewhere already? If not maybe it's worth adding a comment?

Reply:
Oh yes, I see we do that already in very similar situations. Nevermind.